retrieval/target.go

```diff
@@ -538,43 +585,22 @@ func (app relabelAppender) Append(s *model.Sample) error {
 	return app.SampleAppender.Append(s)
 }
 
-// URL returns a copy of the target's URL.
-func (t *Target) URL() *url.URL {
+// BaseLabels returns a copy of the target's base labels.
```

Comment:
Status.html needs that change then too (which will also make it more readable imho).